framework/test/jest/unit/specs/application/node/forger/forging_specs.js

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
can we rename this file to `forging_fixtures.js`? `forging_specs` and `forging.spec` are bit confusing

Reply:
Sure.